praw/models/reddit/live.py

```diff
@@ -37,3 +60,73 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
 
     def _info_path(self):
         return API_PATH['liveabout'].format(id=self.id)
+
+    def updates(self, **generator_kwargs):
+        """Return a :class:`.ListingGenerator` yields :class:`.LiveUpdate` s.
+
+        :param generator_kwargs: keyword arguments passed to
+            :class:`.ListingGenerator` constructor.
+        :returns: A :class:`.ListingGenerator` object which yields
+            :class:`.LiveUpdate` object.
+        """
+        url = API_PATH['live_updates'].format(id=self.id)
+        for update in ListingGenerator(self._reddit, url,
+                                       **generator_kwargs):
+            update._thread = self
+            yield update
+
+
+class LiveUpdate(RedditBase):
+    """An individual :class:`.LiveUpdate` object."""
+
+    STR_FIELD = 'id'
+
+    @property
+    def thread(self):
+        """Return :class:`.LiveThread` object the update object belongs to."""
+        return self._thread
+
+    def __init__(self, reddit, thread_id=None, update_id=None, _data=None):
+        """Initialize a lazy :class:`.LiveUpdate` instance.
+
+        Either ``thread_id`` and ``update_id``, or ``_data`` must be
+        provided.
+
+        .. warning:: At this time, accesing lazy attributes, whose value
+           have not loaded, raises ``AttributeError``.
+
+        :param reddit: An instance of :class:`.Reddit`.
+        :param thread_id: A live thread ID, e.g., ``'ukaeu1ik4sw5'``.
+        :param update_id: A live update ID, e.g.,
+            ``'7827987a-c998-11e4-a0b9-22000b6a88d2'``.
+
+        Usage:
+
+        .. code-block:: python
+
+           update = LiveUpdate(reddit, 'ukaeu1ik4sw5',
+                               '7827987a-c998-11e4-a0b9-22000b6a88d2')
+           update.thread     # LiveThread(id='ukaeu1ik4sw5')
+           update.id         # '7827987a-c998-11e4-a0b9-22000b6a88d2'
+           update.author     # raise ``AttributeError``
+        """
+        if _data is not None:
+            super(LiveUpdate, self).__init__(reddit, _data)
+        elif thread_id and update_id:
+            super(LiveUpdate, self).__init__(reddit, None)
+            self._thread = LiveThread(self._reddit, thread_id)
+            self.id = update_id  # pylint: disable=invalid-name
+        else:
+            raise TypeError('Either `thread_id` and `update_id`, or '
+                            '`_data` must be provided.')
+
+    def __getattr__(self, attribute):
```

Comment:
I removed _fetch() hack. Now accesing missing attributes throw AttributeError. Is this appropriate?